packages/epetra/src/Epetra_IntMultiVector.h

```diff
@@ -706,13 +708,15 @@ class EPETRA_LIB_DLL_EXPORT Epetra_IntMultiVector: public Epetra_DistObject, pub
   int ResetView(int ** ArrayOfPointers);
 
   //! Get pointer to MultiVector values
-  int* Values() const {return Values_;};
+  int* Values() const {return Values_;}
 
   //! Get pointer to individual vector pointers
-  int** Pointers() const {return Pointers_;};
+  int** Pointers() const {return Pointers_;}
   //@}
 
   // Expert-only function
+  //  SuperLU defines Reduce to be a macro in util.h
+#undef Reduce
```

Comment:
You might want to add `#undef Reduce` right below the include for that SuperLUDist header file so that other' code can avoid this pit-fall in the future.